src/components/product-tile/ProductTile.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+import { Product } from "../../types/product";
+
+const ProductTileContainer = styled.li`
+  width: 100%;
+
+  & img {
```

Comment:
it's not good practice to style elements, I guess you've done this so it can be improved later?